website/client/components/groups/chat.vue

```diff
@@ -187,7 +187,7 @@
     position: relative;
 
     textarea {
-      height: 150px;
+      min-height: 150px;
```

Comment:
I checked `min-height` and `height` it doesn't seem to change anything? I can still resize the textarea

Reply:
on my first try it doesn't seemed to use the `min-height`, retesting it again, worked, maybe some browser hiccup